__tests__/functional/register.test.js

```diff
@@ -0,0 +1,123 @@
+import AppRunner from "./utils/AppRunner.js";
+import {
+    expectNoErrors,
+    expectAnyError,
+    expectAnyErrorMessageToBe,
+} from "./utils/matchers.js";
+
+let SUT = new AppRunner(["users"]); //collection 'users' will be emptied after every test
```

Comment:
could be done without comments
```suggestion
collectionsToClean  = ["users"];
let SUT = new AppRunner(collectionsToClean);
```
also, i have no idea what SUT stands for

Reply:
System Under Test ;)
I wrote these comments because I wanted some clear examples I could give to people on how to write such tests. But of course your way is clearer and more universal.